authentication/identity_providers/configuring-basic-authentication-identity-provider.adoc

```diff
@@ -6,9 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Configure a `basic-authentication` identity provider for users to log in to
-{product-title} with credentials validated against a remote identity provider.
```

Comment:
I am not sure long lines are a step in the right direction.

Reply:
> I am not sure long lines are a step in the right direction.

@bergerhoffer I also wonder. To be focused on reviewing technical changes, mixing non-technical changes, frankly speaking, brings inconvenience for us to focus on tech review. Because we need to read the short lines VS long line changes to review to ensure it is purely line length change or it includes tech change for review :) .

Reply:
@xingxingxia This is something that makes it easier on us as a docs team. For example, searching using grep, or sed to find/replace phrases in the entire doc set in one sweep. If we break things up across multiple lines, we can't do that automatic replacement. We also had a lot of manual fixing when we used to auto wrap at 80 character, on certain things like our links/xrefs. So the wrapping works best for us in docs.

I'm sorry that this bothers you both, but I think that this happens infrequent enough that it's not normally an issue during reviews. So it might happen here and there, but I will try to limit those occurrences and will try to separate non-technical changes when I can. 

However, as I mentioned to Standa, to work around this for this PR, I did create this comment: https://github.com/openshift/openshift-docs/pull/39594#issuecomment-1307388815. Can you please use this to do the review/verification?

That contains the list of file diffs and previews for only the new/technical changes. So for this instance, you can focus on just those files and ignore the rest in the PR. Again, I'll try to separate out those non-technical changes as much as possible in the future.

Reply:
Got you. Thanks!

Reply:
Though, to be assured of what are exactly changed, I still went through https://github.com/openshift/openshift-docs/pull/39594/files as well as comparing the previews you gave, otherwise I worry probably not all of the exact changes are checked, if only viewing the previews pages.
/lgtm
Only 2 minor comments.